### ISSUE TITLE
Preference to not close the Arduino IDE when last Sketch is closed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [shovon134/arduino#983]
```
It would be really nice if there was a preference so that the Arduino IDE did 
not close when you are closing the last open Sketch.....

Every IDE I have used stays open when projects are closed - including the last 
one.....  the idea being that you can "close" one project before starting the 
next.

I know this sounds nit-picky....... but it really causes a loss of focus when 
moving from one project to another.... i.e. in order to start another project 
you have to open it.... then move backwards and close the previous one...
```

Original issue reported on code.google.com by `tjanddav...@gmail.com` on 14 Jul 2012 at 11:51
- Merged into: #936
